packages/clis/generator/graph/find-route-worker-wrapper.js

```diff
@@ -0,0 +1,32 @@
+import { workerData as tinypoolData } from 'node:worker_threads';
+import { register } from 'tsx/esm/api';
+
+// This file exists so that:
```

Comment:
N.B.: this file may end up becoming a generic worker wrapper, if more `generator` commands end up being made multi-threaded (e.g., [contour generation](https://github.com/truckermudgeon/maps/blob/7149d69d1871aa52638610c248d6511eaf76836c/packages/clis/generator/geo-json.ts#L808-L822)).